.storybook/with-reduced-motion.tsx

```diff
@@ -0,0 +1,63 @@
+import { Decorator } from "@storybook/react";
+
+declare global {
```

Comment:
question: does this need to be an additional property on the global window? Could we get a way with something like `let originalMatchMedia`?